subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_utils_con_management.c

```diff
@@ -57,6 +57,41 @@ static void lwm2m_utils_dtls_load(struct lwm2m_ctx *ctx)
 	dtls_stored = false;
 }
 
+static void lwm2m_utils_update_rptau(struct lwm2m_ctx *client)
+{
+	int ret;
+	uint32_t lifetime;
+	struct lte_lc_psm_cfg psm_cfg = {0};
+
+	ret = lte_lc_psm_get(&psm_cfg.tau, &psm_cfg.active_time);
+
+	if (ret) {
+		LOG_ERR("Failed to get PSM information");
+		return;
+	}
+
+	if (psm_cfg.active_time < 0) {
+		LOG_DBG("PSM deactivated");
+		return;
+	}
+
+	if (lwm2m_get_u32(&LWM2M_OBJ(1, client->srv_obj_inst, 1), &lifetime) < 0) {
+		lifetime = CONFIG_LWM2M_ENGINE_DEFAULT_LIFETIME;
+		LOG_INF("Using default lifetime: %u", lifetime);
```

Comment:
```suggestion
		LOG_WRN("Using default lifetime: %u", lifetime);
```

I assume that this should not happen usually.